src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,125 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    protected ArrayList<Task> taskLists;
+
+    // divider
+    private final String LINES = "    ---------------------------------";
+
+    /**
+     * Constructs a TaskList object.
+     *
+     * @param taskList an array of Task.
+     */
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskLists = taskList;
+    }
+
+    /**
+     * Returns a size integer of TaskList object.
+     *
+     * @return size of TaskList object.
+     */
+    public int size() {
+        return taskLists.size();
+    }
+
+    /**
+     * Returns a Task object at that TaskList position.
+     *
+     * @param num position to get task.
+     * @return
+     */
+    public Task get(int num) {
+        return taskLists.get(num);
+    }
+
+    /**Lists and print the all the Task in the TaskList.*/
+    public void list() {
+        System.out.println(LINES);
+        System.out.println("    Here are the tasks in your list:");
+        for (int i = 0; i < taskLists.size(); i++) {
+            String display = String.format("    %d.%s", i + 1, taskLists.get(i).toString());
+            System.out.println(display);
+        }
+        System.out.println(LINES);
+    }
+
+    /**
+     * Removes task from TaskList.
+     *
+     * @param num position to remove task.
+     */
+    public void removeTask(int num) {
+
+        System.out.println(LINES);
+        System.out.println("    Noted. I've removed this task:");
+        System.out.println("        " + taskLists.get(num - 1).toString());
+        taskLists.remove(num - 1);
+
+        String taskRemainingString = String.format("    Now you have %d tasks in the list.", taskLists.size());
```

Comment:
A line break could be made here after the comma 👍

Reply:
Agreed, might make the code more readable